server/channels/app/platform/config.go

```diff
@@ -143,19 +145,26 @@ func (ps *PlatformService) ConfigureLogger(name string, logger *mlog.Logger, log
 		if err != nil {
 			return fmt.Errorf("invalid config source for %s, %w", name, err)
 		}
-		ps.logger.Info("Loaded configuration for "+name, mlog.String("source", dsn))
+		resultMsg = fmt.Sprintf("Loaded advanced logging configuration for %s: %s", name, string(dsn))
 	} else {
-		ps.logger.Debug("Advanced logging config not provided for " + name)
+		resultLevel = mlog.LvlDebug
+		resultMsg = fmt.Sprintf("Advanced logging config not provided for %s", name)
 	}
 
 	cfg, err := config.MloggerConfigFromLoggerConfig(logSettings, logConfigSrc, getPath)
 	if err != nil {
 		return fmt.Errorf("invalid config source for %s, %w", name, err)
 	}
 
+	// this will remove any existing targets and replace with those defined in cfg.
 	if err := logger.ConfigureTargets(cfg, nil); err != nil {
 		return fmt.Errorf("invalid config for %s, %w", name, err)
 	}
+
+	if resultMsg != "" {
```

Comment:
This is now output after the logger is configured.  Previous to this change, there was a edge case where if only advanced logging was configured (no default console or file) then log items from this method would be emitted before any log targets were defined.

Reply:
Good catch :+1: